dcpy/extract/templates/bpl_libraries.yml

```diff
@@ -0,0 +1,39 @@
+name: bpl_libraries
+acl: public-read
+source:
+  url: https://www.bklynlibrary.org/locations/json
+
+library_dataset:
```

Comment:
Added a comment in the class definition, but figured easiest thing for now is to include the entire "old" template as a subfield, so we can add/delete freely in the actual new template but have the old ones here for reference

Reply:
Nice solution! I didn't realize we are using "production" templates to build new data library :) Questions:
* It looks like our original templates have `dataset` not `library_dataset` object. Does new object name work with our current data library infra?
* `transform_to_parquet_metadata` <--- what type of data you are thinking of here?

Reply:
1st point - It doesn't. these are not really meant to be used by anything in code, just to live in the files so that we can reference them. As in, as we're building these new ones, make sure that we're not forgetting anything. Once we're done with this project, we can remove this field. If we really wanted to point data library at these for some reason, it would work with very minimal tweaks.

For your second point, not sure - just wanted to have a field that is self contained for #631 , so that additions can be made without collisions from work on #499. Presumably file type might be needed - for a txt within a zip for dcp pad for example. But just wanted a dedicated place for anything needed in implementation of #631. Can merge these or drop the latter later if it's unnecessary.